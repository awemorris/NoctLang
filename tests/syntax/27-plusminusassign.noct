func main() {
     var a = 123;
     a += 1;
     print(a);

     var b = 123;
     b -= 1;
     print(b);

     var c = 123;
     c *= 2;
     print(c);

     var d = 123;
     d /= 2;
     print(d);

     var e = 123;
     e %= 2;
     print(e);

     var f = 123;
     f &= 7;
     print(f);

     var g = 123;
     g |= 127;
     print(g);
}
