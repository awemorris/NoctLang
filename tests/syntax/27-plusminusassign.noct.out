124
122
246
61
1
3
127
